package.json

```diff
@@ -77,7 +77,7 @@
     "cordova-android": "^10.1.1",
     "cordova-browser": "~6.0.0",
     "cordova-ios": "~6.1.0",
-    "cordova-osx": "github:Jigsaw-Code/cordova-osx",
+    "cordova-osx": "github:apache/cordova-osx",
```

Comment:
Do you think maybe it makes sense to pin this to a SHA incase they break something?

Reply:
No need. package-lock.json already does that.

Reply:
I guess the one thing it would guarantee is if we change something related it wouldn't overwrite the existing sha in the package-lock - but I don't see that happeninig